apps/fz_http/test/fz_http_web/live/user_live/show_test.exs

```diff
@@ -340,16 +334,18 @@ defmodule FzHttpWeb.UserLive.ShowTest do
       assert html =~ "120"
     end
 
-    test "prevents empty names", %{admin_conn: conn, admin_user: user} do
+    test "generates a name when it's empty", %{admin_conn: conn, admin_user: user} do
```

Comment:
@jamilbk When an empty string is used as the device name `Ecto.Changeset` ignores the change as empty (and I think it does the right thing). Should I change the behavior back to simply return an error on an empty string?

Reply:
Yeah I think we can count empty string as blank and therefore autogenerate

Reply:
Though I guess it might be a little odd UX to supply an empty name via API and it comes back populated

Reply:
Yeah, I was thinking the same but decided to bring it up because it's a default Ecto behavior (means other projects do the same) and if we want to threat empty string as changed value - we should make sure it's like that in every changeset. WDYT? Simplicity vs API weirdness?

Reply:
Let's go with simplicity! This will probably be changing soon with client apps anyway. We've had pilot feedback to have the device name be generated based on some user/deviceOS/version convention, so the client would presumably provide this in the future.